Core/Resource/SymbolJson.cs

```diff
@@ -255,105 +252,76 @@ private static void ReadChildInputValue(SymbolChild symbolChild, JToken inputJso
 
         private static void ReadChildOutputData(SymbolChild symbolChild, Guid outputId, JToken json)
         {
-            if (json["Type"] != null)
+            if (json[JsonKeys.Type] != null)
             {
                 symbolChild.Outputs[outputId].OutputData.ReadFromJson(json);
             }
         }
 
-        public Symbol ReadSymbol(Model model)
+        public static bool TryReadSymbol(Model model, JToken jToken, out Symbol symbol, bool allowNonOperatorInstanceType)

```

Comment:
the public facing read - currently only used by the `Graph` class I think for pasting operators.

Reply:
LOL I think I'm with you in that I feel like it shouldn't be necessary - we should just be able to "copy/paste" references to pointers. Problem for another day I suppose